proto/noble/dollar/vaults/v2/events.proto

```diff
@@ -294,3 +294,242 @@ message EventVaultConfigUpdated {
     (gogoproto.stdtime) = true
   ];
 }
+
+// EventInflightFundCreated is emitted when an inflight fund operation is initiated
+message EventInflightFundCreated {
+  // Transaction ID
+  string transaction_id = 1;
+
+  // Route ID
+  uint32 route_id = 2;
+
+  // Operation type (DEPOSIT_TO_POSITION, WITHDRAWAL_FROM_POSITION, etc.)
+  string operation_type = 3;
```

Comment:
didn't we decide to use an enum for the operation type in the previous PR dealing with protos?